### PR TITLE
[Fix-10400]Fix K8s deployment bitnami version error

### LINKED ISSUE #10400: Deploy kubernetes according to the official website document, failed!
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

In the official documents of 2.0.5 and 3.0.0-beta-1, there are problems in the deployment of kubernetes. The deployment cannot succeed according to the instructions in the documents.

Encountered the following problems:

1) Execute "helm dependency update." "Error: can't get a valid version for repositories PostgreSQL, zookeeper. try changing the version constraint in chart.yaml". Set chart In yaml, the PostgreSQL version is changed to 11.0.0, and the zookeeper version is changed to 9.0.0. This step can be successful.

2) After "helm install dolphin scheduler. --set image.tag=2.0.5" is executed, the status of pod is createcontainerconfigerror, and the error message of pod is: "error: could't find key PostgreSQL password in secret default/dolphin scheduler PostgreSQL". Check the secret. There is PostgreSQL password in the secret.

### What you expected to happen

I hope the application can be successfully deployed according to the official website documents, but there are many problems.

### How to reproduce

Just follow the official website documents to reproduce the problem.

### Anything else

_No response_

### Version

3.0.0-beta-1

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by qingwli**
- close #10400

### CODE REVIEW

File: deploy/kubernetes/dolphinscheduler/Chart.yaml [hunk 1]
```diff
@@ -43,10 +43,10 @@ appVersion: dev-SNAPSHOT
 
 dependencies:
 - name: postgresql
-  version: 11.6.6
-  repository: https://charts.bitnami.com/bitnami
+  version: 10.3.18
+  repository: https://raw.githubusercontent.com/bitnami/charts/archive-full-index/bitnami
```
Comment (zhongjiajie) [inline deploy/kubernetes/dolphinscheduler/Chart.yaml:52]: should we add some comments to desc why we add this change, I think we should at less point to the issue https://github.com/bitnami/charts/issues/10539

Comment (qingwli) [reply deploy/kubernetes/dolphinscheduler/Chart.yaml:52]: Yes, I agree with you, or the user may be very confused about why we use this URL.

Comment (qingwli) [reply deploy/kubernetes/dolphinscheduler/Chart.yaml:52]: They have a FAQ about this question. Do you think https://github.com/bitnami/charts/issues/10833 is better?

Comment (qingwli) [reply deploy/kubernetes/dolphinscheduler/Chart.yaml:52]: Done

**Review by kezhenxu94 — APPROVED** [review]
LGTM

**Comment by qingwli** [comment]
PTAL @zhongjiajie

**Review by SbloodyS — APPROVED** [review]
+1

**Comment by qingwli** [comment]
This PR impacted K8s deployed, need to change the milestone to 300b2. Thx @SbloodyS

**Comment by SbloodyS** [comment]
> This PR impacted K8s deployed, need to change the milestone to 300b2. Thx @SbloodyS

PTAL @devosend

**Comment by zhongjiajie** [comment]
> This PR impacted K8s deployed, need to change the milestone to 300b2. Thx @SbloodyS

Agree with that, I change milestone to 300b2, and FYI @SbloodyS @devosend

**Pull request merged**